admin-dev/themes/new-theme/scss/theme.scss

```diff
@@ -87,3 +87,6 @@ $color-icons: #6c868e;
     }
   }
 }
+#product_filter_column_active span.select2{
```

Comment:
Please add a space between `select2` and `{` and don't put the `span` as a selector. According to our standards, we shouldn't target directly html element for CSS.